### ISSUE TITLE
Does this code work with IWR6843AOP

### ISSUE THREAD

**Issue by HichriAyoub** [ibaiGorordo/AWR1843-Read-Data-Python-MMWAVE-SDK-3-#35]
Hello,
 i am trying to use this code with IWR6843AOP board with sdk 3.05 the code works fine but gets stuck after sensor start it just reads the config file and sends it and does nothing

**Comment by ibaiGorordo** [comment]
Yes it works, but you need to modify the config file to one that is suitable for the AOP board.

You can get it from the demo visualizer. If the config you create works in the demo visualizer, it should also work here.

**Comment by HichriAyoub** [comment]
somehow the config file works with demo visualiser sdk 3.5 but when try to run range-dopplerheatmap.py the code stop at sensorstart and i printed dataOk variable it's always 0 i think the problem might be coming from the readAndParseData18xx function but i don't know where exactly do you have any idea ?

and i just printed byteBufferlength also and it's always zero might be the problem too because like this it never enters the condition function(if)
